src/plugins/unified_histogram/public/chart/chart.tsx

```diff
@@ -366,12 +366,14 @@ export function Chart({
             data-test-subj="unifiedHistogramRendered"
           >
             {isChartLoading && (
-              <EuiProgress
-                size="xs"
-                color="accent"
-                position="absolute"
-                data-test-subj="unifiedHistogramProgressBar"
-              />
+              <EuiDelayRender delay={500}>
```

Comment:
Nit: Can we leave a comment here describing why we're wrapping this in a delay (basically your comment you had posted)?